### ISSUE TITLE
Remote call failure 

### ISSUE THREAD

**Issue by BridgeAR** [nodejs/build#2088]
All builds where successful but the CI reports an issue due to a remote call failure:

```
00:28:47 FATAL: Channel "unknown": Remote call on JNLP4-connect connection from 58.96.3ca9.ip4.static.sl-reverse.com/169.60.150.88:43072 failed. The channel is closing down or has closed down
00:28:47 java.nio.channels.ClosedChannelException
```

See https://ci.nodejs.org/job/node-test-pull-request/27585/console

**Comment by rvagg** [comment]
@targos mentioned same on IRC yesterday and I saw a string of these last week. Same machine and it looks odd because it comes from the jenkins-workspace machine, i.e. the one that coordinates your parent job(s). I've disabled the machine for now, test-softlayer-ubuntu1604-x64-1, it needs more investigation. Last week I did an update, a slave.jar update, a cleanup, but we're still seeing the same problem. We have two more machines that should pick up the slack just fine while this one's offline: https://ci.nodejs.org/label/jenkins-workspace/
